src/app/components/Transactions/LogEvent.tsx

```diff
@@ -51,7 +51,7 @@ const DecodedLogEvent: FC<{ layer: Layer; event: RuntimeEvent }> = ({ layer, eve
             <dd>
               <b>{event.evm_log_name}</b>
             </dd>
-            {event.evm_log_params?.length && (
+            {event.evm_log_params && event.evm_log_params.length > 0 && (
```

Comment:
Can we keep optional chaining `event.evm_log_params?.length > 0`  to make it shorter?

Reply:
nope, typescript doesn't pass `nullable > number` comparison